.gitlab-ci.yml

```diff
@@ -57,62 +59,43 @@ build:docker:
     - docker
 
 build:make:
-  image: golang:1.11
+  image: docker
+  before_script:
+    - apk update && apk add make
```

Comment:
`- apk add --no-cache make` might be faster on alpine based containers, as it doesn't require the update step - only as a FYI, since you plan to use this as a reference

Reply:
This is the reference to replicate. So yes, thanks!